### PR TITLE
Resource Leak Fix for BufferedInputStream in captureStdErr

### PR DESCRIPTION
**Pull request by Aramismas**

### CODE REVIEW

File: com.ibm.wala.util/src/main/java/com/ibm/wala/util/processes/BasicLauncher.java [hunk 1]
```diff
@@ -10,6 +10,7 @@
  */
 package com.ibm.wala.util.processes;
 
+import java.io.BufferedInputStream;
```
Comment (msridhar) [inline com.ibm.wala.util/src/main/java/com/ibm/wala/util/processes/BasicLauncher.java:13]: This change seems irrelevant?

File: com.ibm.wala.util/src/main/java/com/ibm/wala/util/processes/Launcher.java [hunk 2]
```diff
@@ -178,7 +178,7 @@ void blockingDrain() throws IOException {
     return result;
   }
 
-  protected Drainer captureStdErr(Process p) {
+  protected Drainer captureStdErr(Process p) throws IOException {
```
Comment (msridhar) [inline com.ibm.wala.util/src/main/java/com/ibm/wala/util/processes/Launcher.java:181]: why this change?

**Review by msridhar — CHANGES_REQUESTED** [review]
Let's remove unnecessary changes before I review further

**Comment by msridhar** [comment]
Also, please run `./gradlew googleJavaFormat` and check in the changes, to pass the formatting checks on CI.

File: com.ibm.wala.util/src/main/java/com/ibm/wala/util/processes/Launcher.java (hunk 2 above)
Comment (Aramismas) [reply com.ibm.wala.util/src/main/java/com/ibm/wala/util/processes/Launcher.java:181]: This change was related to the first solution but should be removed as it is irrelevant now.

File: com.ibm.wala.util/src/main/java/com/ibm/wala/util/processes/BasicLauncher.java (hunk 1 above)
Comment (Aramismas) [reply com.ibm.wala.util/src/main/java/com/ibm/wala/util/processes/BasicLauncher.java:13]: I will remove this import.

**Comment by msridhar** [comment]
I believe we have decided these `close()` calls are not necessary, so I am closing out this PR.  Still @Aramismas we appreciate the contribution!
